docs/gui.rst

```diff
@@ -0,0 +1,40 @@
+..  _unpacking-gui:
+
+ReproUnzip GUI
+**************
+
+**reprounzip-qt** is a graphical interface (GUI) for reprounzip, allowing you to unpack and reproduce experiments from ``.rpz`` files without having to use the command-line. You can also set it as the default handler for the ``.rpz`` file extension so you can open them via double-click.
+
+Installation
+============
+
+*reprounzip-qt* comes with the installer on `Windows <https://github.com/ViDA-NYU/reprozip/releases/download/1.0.8/reprounzip-1.0.8-setup.exe>`_ and `Mac <https://github.com/ViDA-NYU/reprozip/releases/download/1.0.8/reprounzip-1.0.8.pkg>`_. If you used one of these, you will be able to double click on any ``.rpz`` file to boot up the GUI. 
```

Comment:
These are links for 1.0.8, which means we'd have to update this page every time we release a new version 😅